### PR TITLE
initital vnet char

### PR DESCRIPTION
**Pull request by jacobcrawford**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: mukube-vnet
+version: "0.1.0"
+maintainers:
+  - name: Jacob Crawford
```
Comment (MartinSchmidt) [inline chart/Chart.yaml:5]: No endline of file!!!!

File: chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+IPALLOC_RANGE_VALUES: 172.30.0.0/16
+weave_kube_image_tag: 2.8.1
+weave_npc_image_tag: 2.8.1
```
Comment (MartinSchmidt) [inline chart/values.yaml:3]: No end line!

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+IPALLOC_RANGE_VALUES: 172.30.0.0/16
```
Comment (MartinSchmidt) [inline chart/values.yaml:1]: values.yaml file naming not consistent! Some are uppercase and some lower

File: chart/templates/vnet_conf.yaml [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+      namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "original-request": {
+              "url": "/k8s/v1.16/net.yaml?k8s-version=Q2xpZW50IFZlcnNpb246IHZlcnNpb24uSW5mb3tNYWpvcjoiMSIsIE1pbm9yOiIyMCIsIEdpdFZlcnNpb246InYxLjIwLjIiLCBHaXRDb21taXQ6ImZhZWNiMTk2ODE1ZTI0OGQzZWNmYjAzYzY4MGE0NTA3MjI5YzJhNTYiLCBHaXRUcmVlU3RhdGU6ImNsZWFuIiwgQnVpbGREYXRlOiIyMDIxLTAxLTEzVDEzOjI4OjA5WiIsIEdvVmVyc2lvbjoiZ28xLjE1LjUiLCBDb21waWxlcjoiZ2MiLCBQbGF0Zm9ybToibGludXgvYW1kNjQifQo=",
```
Comment (MartinSchmidt) [inline chart/templates/vnet_conf.yaml:28]: What is this at the end? secret?

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]
Add missing values.schema.json to validate the values file.

File: chart/templates/vnet_conf.yaml (hunk 4 above)
Comment (jacobcrawford) [reply chart/templates/vnet_conf.yaml:28]: It is the url where the .yaml file was requested. The string is base64 encoding of the k8s-version.

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**